core/sdk-core/src/main/java/software/amazon/awssdk/core/interceptor/SdkInternalExecutionAttribute.java

```diff
@@ -91,6 +96,26 @@ public final class SdkInternalExecutionAttribute extends SdkExecutionAttribute {
      */
     public static final ExecutionAttribute<Boolean> IS_DISCOVERED_ENDPOINT =
         new ExecutionAttribute<>("IsDiscoveredEndpoint");
+    /**
```

Comment:
nit: space before /** (here and below)

Reply:
it was using IDE to move the constant that didn't add the new line. fixed.